aimm_adapters/heald_labview.py

```diff
@@ -0,0 +1,190 @@
+                    meta_dict["UserComment"] = comment_lines
+                elif parsing_case == ParsingCase.scan:
+                    line = " ".join(line.split())  # Remove unwanted white spaces
+                    comment_lines.append(line)
+                    meta_dict["ScanConfig"] = comment_lines
+                elif parsing_case == ParsingCase.amplifier:
+                    comment_lines = line.split("  ")
+                    amplifier_dict = {}
+                    for element in comment_lines:
+                        key, value = element.split(": ", 1)
+                        amplifier_dict[key] = value
+                    meta_dict["AmplifierSensitivities"] = amplifier_dict
+                elif parsing_case == ParsingCase.analog:
+                    comment_lines = line.split("  ")
+                    analog_dict = {}
+                    for element in comment_lines:
+                        key, value = element.split(": ", 1)
+                        analog_dict[key] = value
+                    meta_dict["AnalogInputVoltages"] = analog_dict
+                elif parsing_case == ParsingCase.mono:
+                    comment_lines = line.split("; ")
+                    mono_dict = {}
+                    for element in comment_lines:
+                        key, value = element.split(": ", 1)
+                        mono_dict[key] = value
+                    meta_dict["MonoInfo"] = mono_dict
+                elif parsing_case == ParsingCase.id_info:
+                    comment_lines = line.split("  ")
+                    meta_dict["IDInfo"] = comment_lines
+                elif parsing_case == ParsingCase.slit:
+                    comment_lines = line.split("   ")
+                    meta_dict["SlitInfo"] = comment_lines
+                elif parsing_case == ParsingCase.motor:
+                    comment_lines = line.split("  ")
+                    meta_dict["MotorPositions"] = comment_lines
+                elif parsing_case == ParsingCase.panel:
+                    comment_lines = line.split("; ")
+                    meta_dict["File"] = comment_lines
+                elif parsing_case == ParsingCase.beamline:
+                    meta_dict["Beamline"] = line
+            else:
+                parsing_case = 0
+                continue
+        # Parse data
+        else:
+            line = " ".join(line.split())  # Remove unwanted white spaces
+            sample = line.split()
+            sample = list(map(float, sample))
+            data.append(sample)
+    df = pd.DataFrame(data, columns=headers)
+    return df, meta_dict
+
+
+def build_reader(filepath):
+    "An another approach"
+    with open(filepath) as file:
+        metadata, df = parse_heald_labview(file)
+    return DataFrameAdapter.from_pandas(df, metadata=metadata, npartitions=1)
+
+
+def is_candidate(filename):
+    filename_ext = filename.split(".")
+    if filename_ext[-1].isnumeric():
+        return True
+    return False
```

Comment:
A nice "Pythonic" option here:
```suggestion
    return filename_ext[-1].isnumeric():
```